### PR TITLE
Fix pkg_config sdl2 string

### PR DESCRIPTION
**Pull request by Damirados**
Needed to make this change for it to compile after disabling precompiled dependency.

### CODE REVIEW

File: bundlex.exs [hunk 1]
```diff
@@ -16,7 +16,7 @@ defmodule Membrane.SDL.BundlexProject do
           sdl2: [
             {:precompiled, Membrane.PrecompiledDependencyProvider.get_dependency_url(:sdl2),
              "SDL2"},
-            {:pkg_config, "SDL2"}
+            {:pkg_config, "sdl2"}
```
Comment (mat-hek) [inline bundlex.exs:19]:
```suggestion
            {:pkg_config, "SDL2"},
            {:pkg_config, "sdl2"}
```

I guess the .pc may be named differently on different systems/vendors. Please try leaving them both, if one fails, bundlex should try another

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: bundlex.exs (hunk 1 above)
Comment (Damirados) [reply bundlex.exs]: I have tried it now, but it doesn't seem to work.
```
==> membrane_sdl_plugin
Bundlex: Building natives: player
warning: Bundlex: couldn't load ["SDL2"] libraries with pkg-config due to:
	** (BundlexError) pkg-config error:
	Code: 1
	Package SDL2 was not found in the pkg-config search path.
	Perhaps you should add the directory containing `SDL2.pc'
	to the PKG_CONFIG_PATH environment variable
	Package 'SDL2', required by 'virtual:world', not found
```

Comment (Damirados) [reply bundlex.exs]: Oh, it works it just emits a warning now.

Comment (Damirados) [reply bundlex.exs]: It is [here](https://github.com/membraneframework/bundlex/blob/240f889a26c3e974d3faea36a27849bdd6c27ff2/lib/bundlex/toolchain/common/unix/os_deps.ex#L197).

Comment (mat-hek) [reply bundlex.exs]: Right, I guess we can't get away without it. You can switch the order so the warning will be emitted in the other case :P

Comment (mat-hek) [reply bundlex.exs]: In this case yes, but when fetching a precompiled dep fails and we fall back to pkg-config, we should inform the user what's happening. Maybe we should only warn if the kind of dependency changes, like from pkg-config to precompiled and vice versa?

Comment (Damirados) [reply bundlex.exs]: That seems reasonable, maybe I'll find some time today and try to make PR on bundlex repo.

Comment (Damirados) [reply bundlex.exs]: https://github.com/membraneframework/bundlex/pull/134

**Pull request merged**